lua/bufferline/api.lua

```diff
@@ -189,18 +201,25 @@ end
 function api.goto_buffer_relative(steps)
   render.get_updated_buffers()
 
-  local current = render.set_current_win_listed_buffer()
+  if #state.buffers < 1 then
+    notify('E85: There is no listed buffer', vim.log.levels.ERROR, {title = 'barbar.nvim'})
+    return
+  end
 
-  local idx = utils.index_of(state.buffers, current)
+  local current_bufnr = render.set_current_win_listed_buffer()
+  local idx = utils.index_of(state.buffers, current_bufnr)
 
-  if idx == nil then
-    print("Couldn't find buffer " .. current .. ' in the list: ' .. vim.inspect(state.buffers))
-    return
-  else
-    idx = (idx + steps - 1) % #state.buffers + 1
+  if not idx then -- fall back to: 1. the alternate buffer, 2. the first buffer
+    idx = utils.index_of(state.buffers, bufnr'#') or 1
+    notify(
+      "Couldn't find buffer #" .. current_bufnr .. ' in the list: ' .. vim.inspect(state.buffers) ..
```

Comment:
Could we avoid mix-and-matching single and double quotes? I think we've been using single quotes by convention.

Reply:
I'm certainly willing to convert these to single quotes. Just to be clear, they were converted to double quotes in this specific case because the string contained an asterisk and it seemed easier to read at a glance using doubles because the inner `'` didn't need escaping (i.e. `"Couldn't"` vs `'Couldn\'t'`)